src/main/java/edu/hawaii/its/api/service/MembershipService.java

```diff
@@ -71,6 +81,17 @@ public MembershipResults membershipResults(String currentUser, String uid) {
         // The disjoint of basis plus include and exclude: (Basis + Include) - Exclude
         List<String> groupingMembershipPaths = disjoint(parentGroupingPaths(basisIncludeExcludePaths),
                 parentGroupingPaths(excludePaths));
+        // A list of all group paths, in which the uhIdentifier is listed (including curated groupings), so we can find the intersection with curated groupings
+        List<String> allGroupPaths = groupingsService.allGroupPaths(uid);
+        // The list of all curated groupings
+        List<String> curatedGroupings =
+                grouperService.groupAttributeResults(CURATED)
+                        .getGroupAttributes().stream().map(
+                                GroupAttribute::getGroupPath)
+                        .collect(Collectors.toList());
```

Comment:
This can be a bit more readable like this:
```
 List<String> curatedGroupings =
                grouperService.groupAttributeResults(CURATED).getGroupAttributes().stream()
                                .map(GroupAttribute::getGroupPath)
                                .collect(Collectors.toList());
```